index.html

```diff
@@ -44,7 +44,7 @@
     <div class="shopping-cart">
       <p class="cart-title">Items in Cart</p>
       <div class="cart-list"></div>
-      <p>Total: $<span class="total">0</span></p>
+      <p style="font-weight: bold;">Total: $<span class="total">0</span></p>
```

Comment:
Just wanted to give the *Total *a bold font, didn't want to drag it to the
css. is it bad?

On Wed, Mar 30, 2016 at 10:52 AM, Aaron Hayslip <notifications@github.com>
wrote:

> In index.html
> <https://github.com/Elevationacademy/shopping-cart/pull/5#discussion_r57848640>
> :
>
> > @@ -44,7 +44,7 @@
> >      <div class="shopping-cart">
> >        <p class="cart-title">Items in Cart</p>
> >        <div class="cart-list"></div>
> > -      <p>Total: $<span class="total">0</span></p>
> > +      <p style="font-weight: bold;">Total: $<span class="total">0</span></p>
>
> Why the inline style?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Elevationacademy/shopping-cart/pull/5/files/6587ca0bdc6ed100806fcd623fc31e56877b0a54#r57848640>
>